lib/Models/MapInteractionMode.js

```diff
@@ -23,10 +23,16 @@ function MapInteractionMode(options) {
     this.onCancel = options.onCancel;
 
     /**
-     * Gets or sets the message displayed on the map when in this mode.
+     * Gets or sets the html formatted message displayed on the map when in this mode.
+     * @type {Function}
+     */
+    this.message = function() { return {__html: options.message };};
```

Comment:
Don't put React-isms like this in the Models layer.  Just provide the message as a simple string.